docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Options.java

```diff
@@ -5,25 +5,22 @@
 import static org.keycloak.quarkus.runtime.configuration.MicroProfileConfigProvider.NS_KEYCLOAK_PREFIX;
```

Comment:
Can't we replace this class and `Option` now we have similar types in from the new module?

Reply:
We can replace some of this, unfortunately, looks like `SPI`s configuration is still not covered:
https://github.com/keycloak/keycloak/blob/3ff3aeba29bc4d66d30a7d6243db8ca42ccaf3fa/docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Options.java#L44-L64

In this PR I took out the docs bit to be able to perform the change incrementally (i.e. with subsequent non-breaking PRs)
E.g. the `docs` sub-module will rely on the new configuration only after we move all the configuration keys to the new encoding.